src/Storage/Migration/FunctionsAndProcedures/readinstancefromquery.sql

```diff
@@ -1,4 +1,4 @@
-CREATE OR REPLACE FUNCTION storage.readinstancefromquery_v3(
```

Comment:
any prior versions of this function that can be dropped with this PR?

Reply:
No. We don't know whether v3 is deployed before this version.

Reply:
TT release has already been created, so we know they will be in separate releases.

Reply:
Ok, done